osmoutils/store_helper_test.go

```diff
@@ -425,9 +407,9 @@ func (s *TestSuite) TestGetFirstValueAfterPrefix() {
 				s.store.Set([]byte(key), []byte(fmt.Sprintf("%v", i)))
 			}
 
-			actualValues, err := osmoutils.GetFirstValueAfterPrefix(s.store, tc.prefix, tc.parseFn)
+			actualValues, err := osmoutils.GetFirstValueAfterPrefixInclusive(s.store, tc.prefix, tc.parseFn)
 
-			if tc.expectedErr {
+			if tc.expectedErr != nil {
 				s.Require().Error(err)
 				s.Require().Equal(tc.expectedValues, actualValues)
```

Comment:
You aren't checking the expected error type here

Reply:
Switched to using `s.Require().ErrorContains(...)`, which should check the error string now